app/services/poi_importer.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class PoiImporter
+  def import_from_csv(path_to_csv, kind)
+    CSV.foreach(path_to_csv) do |row|
```

Comment:
Maybe it's a good idea first to remove all data of given 'kind' and next add new data of this kind?
Preferably using transaction:
```ruby
ActiveRecord::Base.transaction do
  # remove data with given type
  # add datat with given type
end
```

Reply:
Okay, that makes sense. But what if the file contains only new POIs (update file)? I think it's better to compare .csv with the Workshop table.
Only problem is that information about the workshop may be out of date.

Reply:
And it's not SEO-friendly if We want to create information pages for the workshops in future (ID (URL) will be changing).